google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettingsTest.java

```diff
@@ -61,6 +61,7 @@ public void settingsAreNotLostTest() {
     String projectId = "my-project";
     String instanceId = "my-instance";
     String appProfileId = "my-app-profile-id";
+    boolean isRefreshingChannel = true;
```

Comment:
can you please add a small test for the default (unset) refreshingChannel as well and that it is false?

Reply:
added 2 tests for default and false